package.json

```diff
@@ -75,6 +75,9 @@
       },
       "translation": {
```

Comment:
Btw, should I bump the version to 0.1.15 while I'm here?

Reply:
No, let's make that a separate standalone commit.